lib/Makefile

```diff
@@ -207,6 +207,7 @@ install: lib liblz4.pc
 	$(INSTALL_DATA) lz4.h $(DESTDIR)$(includedir)/lz4.h
 	$(INSTALL_DATA) lz4hc.h $(DESTDIR)$(includedir)/lz4hc.h
 	$(INSTALL_DATA) lz4frame.h $(DESTDIR)$(includedir)/lz4frame.h
+	$(INSTALL_DATA) lz4file.h $(DESTDIR)$(includedir)/lz4file.h
```

Comment:
My understanding is that you wanted to provide this additional header file in a separate PR ?

Reply:
That was an request from @t-mat. Anyway this change is wrong -- `lz4file.h` should be installed together with `lz4frame_static.h`while here I put it with all the base API.